src/SDKs/Azure.Base/data-plane/Azure.Base/Diagnostics/HttpPipelineEventSource.cs

```diff
@@ -27,9 +27,11 @@ internal sealed class HttpPipelineEventSource : EventSource
         private const int RequestContentEvent = 2;
         private const int ResponseEvent = 5;
         private const int ResponseContentEvent = 6;
+        private const int ResponseContentBlockEvent = 11;
```

Comment:
Any chance that we can expose these in a way that the tests can reference them by name rather than having to hardcode the magic number?

Reply:
The intent here is to assert that event ids do not change as they are part of "public surface" of EventSource in the same way event name is.  This is why I prefer using their numeric representation vs having a shared constant.

Reply:
I get that.  You don't necessarily have to throw a shared constant out.   A test-specific copy/paste would be good enough.   Yes, they could drift over time, but the same could be said if the private members in the class change.    It's not so much a data sharing thing that I'm poking at, I'm more focused on the readability of the tests for those that aren't familiar with the implementation.

Reply:
Will do